app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+
+    # raw_task = params[:task]
+    task = Task.new
+    # task.name = raw_task[:name]
+    # task.description = raw_task[:description]
+    # task.completion_date = raw_task[:completion_date]
+    task.assign_attributes(task_params)
```

Comment:
Its more common to pass the params into the new rather than also assigning the attributes: `Task.new(task_params)`